src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,78 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+
+  const pagesList = [];
+
+  for (let i = 1; i <= totalPages; i++) {
+    pagesList.push(i);
+  }
+
+  const showPages = pagesList.map(page => (
+    <li
+      key={page}
+      className={cn('page-item', { active: currentPage === page })}
+    >
+      <a
+        data-cy="pageLink"
+        className="page-link"
+        href={`#${page}`}
+        onClick={() => onPageChange(page)}
```

Comment:
Using `href` with hash can lead to unwanted scroll behavior. Consider using `button` elements instead of anchor tags or preventing the default action in the `onClick` handler.